libs/ui-toolkit/src/utils/intent.ts

```diff
@@ -9,6 +9,13 @@ export enum Intent {
   Help = 'help',
 }
 
+export enum Variant {
```

Comment:
I think we should probably just use Intent and not add a new Variant enum (if possible) as they both attempt to solve the same problem? We might need to consolidate how we use the intent enum first though so this could be fixed as an interation

Reply:
Hmmm yes I agree, we have variants that are different for button and lozenge, would be good to unify these